jerry-core/ecma/base/ecma-alloc.h

```diff
@@ -30,108 +30,108 @@
  *
  * @return pointer to allocated memory
  */
-extern ecma_object_t *ecma_alloc_object (void);
+extern ecma_object_t * ecma_alloc_object (void);
```

Comment:
@galpeter, @egavrin, what style do you prefer for the following declaration:
`extern ecma_number_t * __attr_pure___ ecma_get_number_from_value (ecma_value_t);`?

Is `extern ecma_number_t *__attr_pure___ ecma_get_number_from_value (ecma_value_t);` version ok for you?